workspace/packages/local/Deck/classic/src/view/main/Main.js

```diff
@@ -2,6 +2,7 @@ Ext.define('Deck.view.main.Main', {
     xtype: 'deck-main',
     extend: 'Ext.panel.Panel',
     requires: [
+        'Ext.plugin.Viewport',
```

Comment:
There's a known issue where this plugin auto dependency is not correctly wired when building. We'll fix that on 6.5.x+ for the time being you need to explicitly require it.